_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+> have to add a documentation section with a description, authors and
+> maintainer, otherwise the recipe will fail.
+>
+> Use the following dataset keys:
+>
+> - project: OBS6
+> - dataset: FLUXCOM
+> - type: reanaly
+> - version: ANN-v1
+> - mip: Lmon
+> - start_year: 2000
+> - end_year: 2000
+> - tier: 3
+>
+> Some of these dataset keys are further explained in the callout boxes in this
+> episode.
+>
+> > ## Answer
+> >
+> > Here's an example recipe
+> >
+> > ```yaml
+> > documentation:
+> >
+> >   description: Test recipe for FLUXCOM data
+> >
+> >   authors:
+> >     - kalverla_peter
+> >
+> >   maintainer:
+> >     - kalverla_peter
+> >
+> > datasets:
+> >   - {project: OBS6, dataset: FLUXCOM, mip: Lmon, tier: 3, start_year: 2000, end_year: 2000, type: reanaly, version: ANN-v1}
+> >
+> > diagnostics:
+> >   FLUXCOM:
+> >     description: Check that ESMValTool can load the cmorized FLUXCOM data without errors.
+> >     variables:
+> >       gpp:
+> >       gppStderr:
+> >     scripts: null
+> >
+> > ```
+> >
+> > Note: a recipe similar to this one is available under
+> > `~/path/to/ESMValTool/esmvaltool/recipes/examples/recipe_check_obs.yml`.
+> > That recipe includes checks all datasets for which CMORizers are available.
+> >
+> {: .solution}
+{: .challenge}
+
+Try to run the example recipe with
+
+```bash
+esmvaltool run recipe_check_fluxcom.yml --log_level debug
+```
+
+The `log_level` flag ensures that all relevant information is included in the
+output. We will need this information to find out what needs to be done. Look
+carefully through the log messages. You'll probably find something like
+
+```
+DEBUG   Retrieving OBS6 configuration
+DEBUG   Skipping non-existent /home/peter/default_inputpath/Tier3/FLUXCOM
+DEBUG   Looking for files matching ['OBS6_FLUXCOM_reanaly_ANN-v1_Lmon_gppStderr[_.]*nc'] in []
+ERROR   No input files found for variable {'variable_group': 'gppStderr' ...
+...
+ERROR   Missing data for preprocessor FLUXCOM/gppStderr
+...
+ERROR   Program terminated abnormally, see stack trace below for more information:
+...
+```
+{: .error}
+
+So the output tells us that it cannot find the data, and also where it has been
+looking. Let's try to understand what's going on.
+
+## Location, location, location
+
+The error messages above tell you that ESMValTool cannot find the data. That
+makes total sense, because we have not yet created our CMORized copy of the
+data. Our data is located in the `RAWOBS` folder, but ESMValTool is looking in
+`OBS6`. So the first thing our CMORizer will need to do is copy the data over
+from one folder to the other. To do end, we need to tell ESMValTool where our
+data may be found.
+
+> ## Set the correct paths in your user configuration file:
+>
+> This information is set in `config-user.yml`. Modify your
+> configuration file so that it has the correct paths
+>
+> ```yaml
+> rootpath:
+>   OBS6: /path/to/my/obs6/data
+>   RAWOBS: /path/to/my/rawobs/data
+> ```
+{: .challenge}
```

Comment:
Why the instruction about setting the correct path is in a {:challenge} box while no solution is provided for this challenge? Also, we first teach something and then design a challenge. So users can practice what they have already learned. I suggest making the instruction plain text.

Reply:
Sorry I thought the challenge was also meant to distinguish exercise from theory. I've changed it to plain text.

Reply:
It is right that a challenge is "meant to distinguish exercise from theory". However, setting the correct path (here) and downloading data (in the above) are more **instructions** rather than **exercises**. 
A challenge provides learners an opportunity to practice what that you’ve included in the skill list (objectives) for designing the lesson. In designing a challenge, we have to first teach something then challenge learners if they can do it. I hope now it is clear why I asked to convert those instructions to plain texts.